client/reader/list-manage/index.jsx

```diff
@@ -136,6 +179,15 @@ function ReaderListEdit( props ) {
 	);
 }
 
-export default function ReaderListManage( props ) {
+function ReaderListManage( props ) {
 	return props.isCreateForm ? ReaderListCreate() : ReaderListEdit( props );
 }
+
+export default connect( ( state, ownProps ) => {
```

Comment:
we don't need this at all if we're using `useSelector` in the correct place

Reply:
actually we do. hooks have to be called every time we render, in the same order. we're calling this selector conditionally, so we have to bake it into connect.